src/test/regress/sql/multi_extension.sql

```diff
@@ -600,13 +600,26 @@ RESET client_min_messages;
 
 SELECT * FROM multi_extension.print_extension_changes();
 
--- Test downgrade to 11.2-1 from 11.3-1
+-- Snapshot of state at 11.2-1
```

Comment:
This portion of the file is intended to cover downgrades, and then upgrades. Tests for the last versions are partially broken and it is not so clear for authors to move forward here.

For example, here is a clear section that documents the changes in a cluster that was at `10.1-1` previously:
```
-- Test downgrade to 10.1-1 from 10.2-1
ALTER EXTENSION citus UPDATE TO '10.2-1';
ALTER EXTENSION citus UPDATE TO '10.1-1';
-- Should be empty result since upgrade+downgrade should be a no-op
SELECT * FROM multi_extension.print_extension_changes();

-- Snapshot of state at 10.2-1
ALTER EXTENSION citus UPDATE TO '10.2-1';
SELECT * FROM multi_extension.print_extension_changes();
```

Here is what we test:
1. When we perform an upgrade and a downgrade, the objects remain the same on the schema. That means that the contents of the downgrade files look sane.
2. After the upgrade, we print the changes to the schema and document the objects that changed in this schema version.
3. If we want to test some other thing in version `10.2-1` we do it after this block I shared above.

For example, we do not want to do the following:
```sql
-- Test downgrade to 11.2-2 from 11.3-1
ALTER EXTENSION citus UPDATE TO '11.2-2';
SELECT * FROM multi_extension.print_extension_changes();
```

We want to focus on schema changes on upgrades. If the downgrade operation reverts all the changes to the schema, then it is ok. There is no need to document the changes in the schema after a downgrade.